parsers/mysql/mysql.go

```diff
@@ -307,19 +312,19 @@ func (p *Parser) handleEvents(rawEvents <-chan rawEvent, send chan<- event.Event
 func (p *Parser) handleEvent(rawE rawEvent) SlowQuery {
 	var err error
 	sq := SlowQuery{}
-	logrus.WithFields(logrus.Fields{
-		"rawE": rawE,
-	}).Debug("About to parse event")
+	query := ""
 	for _, line := range rawE.lines {
 		// parse each line and populate the SlowQuery object
 		switch {
 		case reTime.MatchString(line):
+			query = ""
```

Comment:
ahh, because it's used to build up multi-line queries. But if you start what looks like a multi-line query, then get some other line type, you want to throw out the first part of what looked like the multi-line query?

I think a comment is in order.